core/src/main/scala/tech/beshu/ror/accesscontrol/factory/RawRorConfigBasedCoreFactory.scala

```diff
@@ -378,6 +388,47 @@ class RawRorConfigBasedCoreFactory(rorMode: RorMode)
       decoder.apply(c)
     }
   }
+
+  private def localUsersFromUsernamePatterns(userIdPatterns: UserIdPatterns,
+                                             unknownUsersForWildcardPattern: Boolean): LocalUsers = {
+    userIdPatterns
+      .patterns
+      .map(_.value)
+      .map {
+        case Refined(pattern) if pattern.contains(wildcardChar) =>
```

Comment:
maybe we could have `def containsWIldcard: Boolean` method in `UserIdPattern` class?